apps/daimo-mobile/package.json

```diff
@@ -51,52 +51,52 @@
     "@trpc/client": "^10.43.2",
     "@trpc/react-query": "^10.43.2",
     "@types/jest": "^29.5.2",
-    "@types/react": "^18.2.15",
+    "@types/react": "~18.2.45",
     "babel-plugin-transform-inline-environment-variables": "^0.4.4",
     "cbor": "^9.0.1",
     "cbor-rn-prereqs": "^9.0.0",
     "depcheck": "^1.4.3",
     "eslint": "^8.44.0",
     "eslint-config-universe": "^11.3.0",
-    "expo": "^49.0.7",
-    "expo-application": "^5.1.1",
-    "expo-barcode-scanner": "^12.3.2",
-    "expo-build-properties": "~0.8.3",
-    "expo-clipboard": "~4.3.1",
-    "expo-contacts": "^12.4.0",
-    "expo-crypto": "^12.6.0",
-    "expo-dev-client": "~2.4.8",
-    "expo-device": "^5.2.1",
-    "expo-file-system": "~15.4.5",
+    "expo": "^50.0.11",
+    "expo-application": "~5.8.3",
+    "expo-barcode-scanner": "~12.9.3",
+    "expo-build-properties": "~0.11.1",
+    "expo-clipboard": "~5.0.1",
+    "expo-contacts": "~12.8.2",
+    "expo-crypto": "~12.8.1",
+    "expo-dev-client": "~3.3.9",
+    "expo-device": "~5.9.3",
+    "expo-file-system": "~16.0.8",
     "expo-font": "^11.4.0",
-    "expo-haptics": "~12.4.0",
-    "expo-linking": "~5.0.2",
-    "expo-local-authentication": "~13.4.1",
-    "expo-localization": "^14.3.0",
-    "expo-notifications": "^0.20.1",
-    "expo-splash-screen": "~0.20.5",
-    "expo-status-bar": "~1.6.0",
-    "expo-system-ui": "~2.4.0",
+    "expo-haptics": "~12.8.1",
+    "expo-linking": "~6.2.2",
+    "expo-local-authentication": "~13.8.0",
+    "expo-localization": "~14.8.3",
+    "expo-notifications": "~0.27.6",
+    "expo-splash-screen": "~0.26.4",
+    "expo-status-bar": "~1.11.1",
+    "expo-system-ui": "~2.9.3",
     "jest": "^29.5.0",
-    "jest-expo": "^49.0.0",
+    "jest-expo": "~50.0.3",
     "react": "18.2.0",
     "react-error-boundary": "^4.0.12",
-    "react-native": "0.72.4",
+    "react-native": "0.73.4",
     "react-native-email-link": "^1.15.0",
-    "react-native-gesture-handler": "~2.12.0",
-    "react-native-get-random-values": "~1.9.0",
+    "react-native-gesture-handler": "~2.14.0",
+    "react-native-get-random-values": "~1.8.0",
     "react-native-mmkv": "^2.8.0",
     "react-native-qrcode-svg": "^6.2.0",
-    "react-native-reanimated": "~3.3.0",
-    "react-native-safe-area-context": "4.6.3",
-    "react-native-screens": "^3.27.0",
-    "react-native-shake": "^5.5.2",
-    "react-native-svg": "13.9.0",
+    "react-native-reanimated": "~3.6.2",
+    "react-native-safe-area-context": "4.8.2",
+    "react-native-screens": "~3.29.0",
+    "react-native-shake": "^5.6.0",
+    "react-native-svg": "14.1.0",
     "react-native-url-polyfill": "^2.0.0",
```

Comment:
We might not need this anymore? Acc to the blog https://expo.dev/changelog/2024/01-18-sdk-50

Reply:
ya can try deleting ^